### DISCUSSION TITLE
how to hide dashboard [fullscreen] button

### DISCUSSION THREAD

**Discussion by 1101826140** [apache/superset#25937] [Q&A / Help]
i use iframe tag show charts , url is :  https://xxxx/superset/dashboard/xxxx/?standalone=1&expand_filters=1 

and hide header bar ，This is the result I want.，  but i can't hide  [fullscreen] button

when i click dashboard fullscreen button ,and exit fullscreen button, show header bar

now ,  I want to know how to hide button by role， or click exit fullscreen button, hide header bar

**Comment by sfirke** [comment]
When I set `standalone=2` then I don't see the header bar or the choice to exit fullscreen.  Does that address your problem?

**Reply by 1101826140** [reply]
I think this parameter can avoid the problem, but it is not the best solution， and  now i use css template hide [exit fullscreen] button,but it has brought other issues...

**Comment by sfirke** [comment]
Since this is a question about using Superset, and not a bug report, I'm going to convert it to a Discussion instead of an Issue, which is used by the project for tracking bugs.

**Comment by rushirao17** [comment]
In our case, we needed to hide the Toggle Fullscreen button for non-admin users.
We did it by modifying the React code to include an isAdmin check, and then rebuilding the Superset image.
File Location: superset-frontend/src/dashboard/components/Header/useHeaderActionsDropdownMenu.tsx


const user = useSelector<any, UserWithPermissionsAndRoles | undefined>(
  state => state.user,
);

const isAdmin = isUserAdmin(user);

// Modify following snippet
// Only show fullscreen option for admins, and only when not in edit or embedded mode
if (isAdmin && !editMode && !isEmbedded) {
  menuItems.push({
    key: MenuKeys.ToggleFullscreen,
    label: getUrlParam(URL_PARAMS.standalone)
      ? t('Exit fullscreen')
      : t('Enter fullscreen'),
  });
}
